.github/workflows/reproducible-build.yml

```diff
@@ -86,6 +90,42 @@ jobs:
           fetch-depth: 0
           repository: ${{ env.GITHUB_REPOSITORY }}
 
+      - name: Setup Credentials
+        uses: webfactory/ssh-agent@v0.9.0
+        env:
+          SUPER_SECRET: ${{ secrets.deploy-keys }}
+        if: ${{ env.SUPER_SECRET != '' }}
+        with:
+          ssh-private-key: ${{ secrets.deploy-keys }}
+
+      - name: Install rust
+        uses: actions-rust-lang/setup-rust-toolchain@v1
+        env:
+          SUPER_SECRET: ${{ secrets.deploy-keys }}
+        if: ${{ env.SUPER_SECRET != '' }}
+        with:
+          toolchain: ${{ inputs.rust-toolchain }}
+          target: wasm32-unknown-unknown
+
+      - name: Vendored dependencies (if private dependencies are used)
```

Comment:
Thus, private dependencies are brought locally, before starting the build, correct (sorry if I misunderstood)?

If so, the usual _contract verification flow_ (which relies on the Docker container to perform the whole build) would not be able to cover this case (however, for verified contracts, private dependencies would not be acceptable, anyway).